entrypoint.sh

```diff
@@ -1,23 +1,33 @@
 #!/bin/sh
 
 # Generate a unique schema name and user credentials using a timestamp
-SCHEMA_NAME="schema_$(date +%s)"
-NEW_DB_USER="user_$(date +%s)"
+NEW_SCHEMA_NAME="marketplace_squid_$(date +%s)"
+NEW_DB_USER="marketplace_squid_user_$(date +%s)"
```

Comment:
What if we use a shortened version of the commit hash? In that way, we could prevent a re-indexing of the schema whenever the indexer is re-started.